spec/features/editing_images/edit_image_spec.rb

```diff
@@ -2,14 +2,15 @@
 
 RSpec.feature "Edit image", js: true do
   scenario "lead image" do
-    given_there_is_an_edition_with_images
+    given_there_is_an_edition_with_a_lead_image
 
     when_i_visit_the_images_page
     and_i_edit_the_image_crop
     then_the_image_crop_is_updated
 
     when_i_edit_the_image_metadata
     then_i_see_the_image_is_updated
+    and_the_image_is_still_lead_image
```

Comment:
Good idea to have this check. I wonder if we can make it a visual check - we try to avoid DB checks in the features, since the user can't see the DB - by copying/modifying the preview step and using a 'within' block to check the same thing. Something like this:

```
def and_i_see_the_lead_image_is_updated
  within("something around the lead image") { and_j_see_the_image_is_updated }
end
```

Reply:
Yep that makes sense.